chain-core/src/state/tendermint.rs

```diff
@@ -250,6 +250,12 @@ impl From<TendermintVotePower> for u64 {
     }
 }
 
+impl fmt::Display for TendermintVotePower {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0.to_string())
```

Comment:
I don't think you have to manually call `.to_string()` here.

```suggestion
        write!(f, "{}", self.0)
```